src/NServiceBus.Gateway.Tests/ApprovalFiles/APIApprovals.Approve.netframework.approved.txt

```diff
@@ -162,6 +162,12 @@ namespace NServiceBus
         public void Retries(int numberOfRetries, System.TimeSpan timeIncrease) { }
         public void TransactionTimeout(System.TimeSpan timeout) { }
     }
+    public interface IGatewayDeduplicationStorage
```

Comment:
Should be NServiceBus.Gateway according to https://github.com/Particular/PlatformArchitecture/blob/master/documentation/cross-cutting-concepts.md#nservicebus-rules

Reply:
I thought we had some documentation about this but didn't find it at first glance 👍